tools/ja-rule/ja-rule.cpp

```diff
@@ -407,8 +709,20 @@ class InputHandler {
 int main(int argc, char **argv) {
   ola::AppInit(&argc, argv, "[ options ]", "Ja Rule Admin Tool");
 
+  auto_ptr<UID> target_uid(UID::FromString(FLAGS_target_uid));
+  if (!target_uid.get()) {
```

Comment:
Again, it's assumed if you're using this you know what you're doing. I'm trying to limit the tool to printing useful information.